src/components/EstimatedLifecycleCost/index.tsx

```diff
@@ -148,9 +164,11 @@ const EstimatedLifecycleCost = ({
             return (
               <Phase
                 key={`Year1Phase-${index + 1}`}
-                formikKey={`${formikKey}.year1`}
+                formikKey={formikKey}
```

Comment:
Part of my brain wants to create each year programmatically, and the rest of me wonders if this is indeed the correct React way of doing things. @christopherhuii can you add some flavor to why this is the better way?